### PR TITLE
Added Navigation Drawer UI with create feature

### PR DESCRIPTION
**Pull request by surbhicis**
## Repository contributions to the PyBitmessage project

- You can get paid for merged commits if you register at [Tip4Commit](https://tip4commit.com/github/Bitmessage/PyBitmessage)

### Code

- Try to refer to github issue tracker or other permanent sources of discussion about the issue.
- It is clear from the diff *what* you have done, it may be less clear *why* you have done it so explain why this change is necessary rather than what it does

### Documentation

- If there has been a change to the code, there's a good possibility there should be a corresponding change to the documentation 
- If you can't run `fab build_docs` successfully, ask for someone to run it against your branch

### Tests

- If there has been a change to the code, there's a good possibility there should be a corresponding change to the tests
- If you can't run `fab tests` successfully, ask for someone to run it against your branch

## Translations

- For helping with translations, please use [Transifex](https://www.transifex.com/bitmessage-project/pybitmessage/).
- There is no need to submit pull requests for translations.
- For translating technical terms it is recommended to consult the [Microsoft Language Portal](https://www.microsoft.com/Language/en-US/Default.aspx).

### Gitiquette

- Make the pull request against the ["v0.6" branch](https://github.com/Bitmessage/PyBitmessage/tree/v0.6)
- PGP-sign the commits included in the pull request
- Use references to tickets, e.g. `addresses #123` or `fixes #234` in your commit messages
- Try to use a good editor that removes trailing whitespace, highlights potential python issues and uses unix line endings
- If for some reason you don't want to use github, you can submit the patch using Bitmessage to the "bitmessage" chan, or to one of the developers.

### CODE REVIEW

**Comment by g1itch** [comment]
Still black window for me ):

**Comment by surbhicis** [comment]
@g1itch I have to check what dependency its having ,might some python dependencies are missing.will update it soon.

**Comment by PeterSurda** [comment]
It looks like some library is missing, causing gi to crap out, but I can't find which.

**Comment by PeterSurda** [comment]
It looks like kivy included in debian and ubuntu is broken or somehow else incompatible, maybe just too old. If you use kivy from pip, it works (debian 9, ubuntu 16.4 and 18.4). you need to `apt-get install libgl1-mesa-dev cython xsel xclip` to build kivy using pip.

**Comment by surbhicis** [comment]
Kivy, for which OpenGL 2.0 seems to be mandatory as opengl es 2 features that kivy relies on.
Strictly, Kivy targets OpenGL ES 2.0 as the minimum requirement. This is not the same as OpenGL 2.0.
some places you tend to see lack of support are older machines with integrated graphics, Any machine with a 'proper' graphics card, or integrated graphics will almost certainly work fine., if found something like OPENGL 2.0 bug.

Add the following 2 lines to solve OPENGL 2.0 bug

from kivy import Config
Config.set('graphics', 'multisamples', '0') //default its 2

File: src/bitmessagekivy/mpybit.py [hunk 1]
```diff
@@ -61,7 +59,15 @@ def __init__ (self,**kwargs):
...
 
 class Trash(Screen):
-    pass
+    def __init__ (self,**kwargs):
+        super (Trash, self).__init__(**kwargs)
+        val_y = .1
+        val_z = 0
+        my_box1 = BoxLayout(orientation='vertical')
+        for i in range(1, 5):
+            my_box1.add_widget(Label(text="I am in trash", size_hint = (.3,.1), pos_hint = {'x': val_z,'top': val_y},color = (0,0,0,1), background_color = (0,0,0,0)))
```
Comment (omkar1117) [inline src/bitmessagekivy/mpybit.py]: PEP8 standards missing here

Comment (surbhicis) [reply src/bitmessagekivy/mpybit.py]: ok I 'll make it pep8 standarized

File: src/bitmessagekivy/mpybit.py [hunk 2]
```diff
@@ -1,48 +1,117 @@
+import os
 import time
-
+import shutdown
+import queues
```
Comment (omkar1117) [inline src/bitmessagekivy/mpybit.py]: maintain a space in between import and from statements.

File: src/bitmessagekivy/mpybit.py [hunk 3]
```diff
@@ -1,48 +1,117 @@
...
 from kivy.app import App
-from kivy.core.window import Window
+from kivy.lang import Builder
```
Comment (omkar1117) [inline src/bitmessagekivy/mpybit.py]: Al the things should be in a alphabetic order.

Comment (surbhicis) [reply src/bitmessagekivy/mpybit.py]: surely, I'll make them changes.

**Pull request merged**